webapp/src/components/chat/chat-history/ChatHistory.tsx

```diff
@@ -31,17 +31,14 @@ export const ChatHistory: React.FC<ChatHistoryProps> = ({ messages, onGetRespons
 
     return (
         <div className={classes.root}>
-            {messages
-                .slice()
-                .sort((a, b) => a.timestamp - b.timestamp)
```

Comment:
this isn't needed because we already order the messages correctly using the `.reverse()` in `ChatService`